src/main/java/me/partlysanestudios/partlysaneskies/dungeons/partymanager/PartyMember.java

```diff
@@ -407,6 +407,19 @@ private void createMemberBlockColumnTwo(UIComponent memberBlock, float scaleFact
 
     }
 
+    public Color colorFloorRuns(int floorRuns) {
+        if (floorRuns == 0) {
+            return Color.RED;
+        }
+        else if (floorRuns < 10) {
```

Comment:
Could you make this value customisable in the config?

Reply:
Good idea!